crates/esplora/src/async_ext.rs

```diff
@@ -149,7 +129,7 @@ impl EsploraAsyncExt for esplora_client::AsyncClient {
             type IndexWithTxs = (u32, Vec<esplora_client::Tx>);
 
             loop {
-                let futures: FuturesOrdered<_> = (0..parallel_requests)
+                let futures = (0..parallel_requests)
```

Comment:
> it can never be more than 25

I don't think this statement is correct.
https://github.com/Blockstream/esplora/blob/master/API.md#get-scripthashhashtxs

I think we should leave it as is. These are style nits.